src/View/Helper/PaginatorHelper.php

```diff
@@ -712,9 +712,10 @@ protected function _modulusNumbers($templater, $params, $options)
             ]);
         }
 
+        $url = array_merge($options['url'], ['page' => $params['page']]);
```

Comment:
Bunch if tests fail becuase of undefined key. Using isset like @dereuromark suggested will avoid that issue.